src/controller/CHIPDeviceController.cpp

```diff
@@ -915,6 +920,16 @@ void DeviceCommissioner::OnSessionEstablished()
     }
 
     ChipLogDetail(Controller, "Remote device completed SPAKE2+ handshake\n");
+
+    // TODO: Add code to wait for OpCSR from the device, and process the signing request
```

Comment:
Unsure if we should phrase as 'wait' directly as this runs in the main event loop, so wait is likely not possible. We probably need some sort of state machine on what we are sending out/expecting to receive.

Reply:
I have updated the comment to reflect that it's not actually inline wait. It'll receive and process the CSR.